ixnilatisApp/IxnilatisScreens.js

```diff
@@ -22,6 +24,18 @@ export function getIxnilatisScreens(Stack) {
         name="FormGeneralActiveScreen"
         component={FormGeneralActive}
         options={{headerShown:false}}
+      />,
+      <Stack.Screen
+          key="PrivacyScreen"
+          name="PrivacyScreen"
+         component={getPrivacyScreens}
```

Comment:
```suggestion
           component={getPrivacyScreens}
```
notice the styling change, the identation is a bit off

`prettier` should take care of things like this

Reply:
I did not have time to check it out - will do!